tests/tensorflow/quantization/test_builder_state.py

```diff
@@ -148,179 +136,6 @@ def test_checkpoint_callback_make_checkpoints(mocker, tmp_path):
-                "target_type": {
-                    "name": "OPERATOR_POST_HOOK"
-                }
-            },
-            "target_point_class_name": "TFLayerPoint"
-        },
-        {
-            "op_name": "dummy_str",
-            "quantizer_spec": {
-                "half_range": True,
-                "mode": "symmetric",
-                "narrow_range": True,
-                "num_bits": 8,
-                "per_channel": True,
-                "signedness_to_force": None
-            },
-            "target_point": {
-                "layer_name": "dummy_str"
-            },
-            "target_point_class_name": "TFLayer"
-        },
-        {
-            "op_name": "dummy_str",
-            "quantizer_spec": {
-                "half_range": True,
-                "mode": "symmetric",
-                "narrow_range": True,
-                "num_bits": 8,
-                "per_channel": True,
-                "signedness_to_force": None
-            },
-            "target_point": {
-                "in_port": 0,
-                "instance_index": 0,
-                "layer_name": "dummy_str"
-            },
-            "target_point_class_name": "TFBeforeLayer"
-        },
-        {
-            "op_name": "dummy_str",
-            "quantizer_spec": {
-                "half_range": True,
-                "mode": "symmetric",
-                "narrow_range": True,
-                "num_bits": 8,
-                "per_channel": True,
-                "signedness_to_force": None
-            },
-            "target_point": {
-                "instance_index": 0,
-                "layer_name": "dummy_str",
-                "out_port": 0
-            },
-            "target_point_class_name": "TFAfterLayer"
-        },
-        {
-            "op_name": "dummy_str",
-            "quantizer_spec": {
-                "half_range": True,
-                "mode": "symmetric",
-                "narrow_range": True,
-                "num_bits": 8,
-                "per_channel": True,
-                "signedness_to_force": None
-            },
-            "target_point": {
-                "layer_name": "dummy_str",
-                "weights_attr_name": "dummy_str"
-            },
-            "target_point_class_name": "TFLayerWeight"
-        },
-        {
-            "op_name": "dummy_str",
-            "quantizer_spec": {
-                "half_range": True,
-                "mode": "symmetric",
-                "narrow_range": True,
-                "num_bits": 8,
-                "per_channel": True,
-                "signedness_to_force": None
-            },
-            "target_point": {
-                "layer_name": "dummy_str",
-                "operation_name": "dummy_str",
-                "weights_attr_name": "dummy_str"
-            },
-            "target_point_class_name": "TFOperationWithWeights"
-        }
-    ]
-}
-
-
-def test_quantizer_setup_serialization():
-    setup = QuantizationSetup()
```

Comment:
@ljaljushkin note that I removed `QuantizationSetup` altogether in favor of the more general `SingleConfigQuantizerSetup`, please advise on how to better regenerate the checkpoints

Reply:
Luckily, all current sota checkpoints can be restored without loading old builder state. Because there's no unambiguous configurations yet.

So you can comment loading of the builder state and save the new one instead of doing validation within sota eval checkpoint test.

Reply:
Restored QuantizationSetup.